src/Components/Users/UserProfile.tsx

```diff
@@ -450,6 +466,17 @@ export default function UserProfile() {
                           errors={states.errors.phoneNumber}
                         />
                       </div>
+
+                      <div className="col-span-6 sm:col-span-3">
+                        <PhoneNumberField
+                          label="Whatsapp Number*"
+                          value={states.form.altPhoneNumber}
+                          onChange={(value: any) => [
+                            handleValueChange(value, "altPhoneNumber"),
+                          ]}
```

Comment:
```suggestion
                          onChange={(value: any) => {
                            handleValueChange(value, "altPhoneNumber")
                          }}
```

Reply:
I'll do that @smartclash

Reply:
Its failing when I change that. I dont know the reason. @bodhish you wrote that code. Why is it saying expression expected?

Reply:
@peeyush14goyal Not sure. Will have to dig it. It would mostly be a refactor that is showing my name

Reply:
Changed to { } @bodhish @smartclash